internal/validators/time.go

```diff
@@ -0,0 +1,12 @@
+package validators
+
+import (
+	"time"
+
+	"github.com/billz-2/ofd_connector/internal/constants"
+)
+
+func IsValidateTimeFormat(dateTime string) bool {
+	_, err := time.Parse(constants.TimeFormat, dateTime)
+	return err == nil
```

Comment:
What if err is not nil? i bet we could return it? may be we will use (bool, error) return type? for logs.

Reply:
I made to return just error.